src/components/mock/categories.js

```diff
@@ -0,0 +1,8 @@
+import {v4 as uuidv4} from 'uuid' 
+
+export const mockTeachers=[
```

Comment:
@Mariana-Usuga in export const you are using same variable name to all files. Please use names related to the file.

Reply:
Done